doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -287,7 +287,12 @@ Serial LTE modem
 
 * Added an overlay :file:`overlay-memfault.conf` file to enable Memfault.
   For more information about Memfault features in |NCS|, see :ref:`mod_memfault`.
-* Updated the application to use the :ref:`lib_downloader` library instead of the deprecated :ref:`lib_download_client` library.
+
+* Updated:
+
+  * The application to use the :ref:`lib_downloader` library instead of the deprecated :ref:`lib_download_client` library.
+  * In Zephyr, the numerical values of various |NCS| specific socket options that are used with the ``#XSOCKETOPT`` command.
```

Comment:
I think we could also mention the effect this has on the SLM application to be more clear.

Reply:
Do you have a specific suggestion?

Reply:
Discussed offline with Divya